tests/test_prompts.py

```diff
@@ -79,12 +91,36 @@ def test_save_and_load(fixture_foobar_prompt, tmpdir):
     assert FooBarPrompt.load(filepath) == fixture_foobar_prompt
 
 
-def test_messages(fixture_messages_prompt, fixture_expected_messages_prompt_messages):
+def test_messages(
+    fixture_messages_prompt: MessagesPrompt, fixture_expected_messages_prompt_messages
+):
     """Tests that the messages decorator adds a function `messages` attribute."""
     assert hasattr(fixture_messages_prompt, "messages")
     assert fixture_messages_prompt.messages == fixture_expected_messages_prompt_messages
 
 
+def test_tags(fixture_tag_prompt: TagPrompt, fixture_tags_prompt: TagsPrompt):
+    """Tests that the tags decorator adds a `tags` attribute."""
+    assert hasattr(fixture_tag_prompt, "_tags")
+    assert fixture_tag_prompt._tags == ["test_tag"]
+    assert fixture_tags_prompt._tags == ["multiple", "tags"]
+
+
+def test_prompt_dump(
```

Comment:
you can/should parameterize this to run for multiple different prompts, as follows:

```python
@pytest.mark.parametrize("prompt", ["fixture_foobar_prompt", "fixture_messages_prompt"])
def teset_prompt_dump(
    prompt, fixture_chat_completion, request
):
    prompt = request.getfixturevalue(prompt)
    ...
```

You can do the same for the above `test_tags` function, just include an additional `expected_tags` parameter that you test against for each `prompt` parameter

Reply:
since i moved completion dump outside of test_prompts, i dont need it for test_prompt_dump anymore but i implemented it for tags